packages/shared/src/components/auth/PostAuthBanner.tsx

```diff
@@ -15,9 +38,30 @@ export const PostAuthBanner = (): ReactElement => {
     shouldEvaluate: isCompatibleBrowser,
   });
 
+  // const { value: showPersonalizedBanner } = useConditionalFeature({
+  //   feature: feature.postPersonalizedBanner,
+  //   shouldEvaluate: isCompatibleBrowser,
+  // });
```

Comment:
Since we are almost at the end stages of this, we can now revert the changes on this part.